addons/website/static/src/snippets/s_progress_bar/options.js

```diff
@@ -42,7 +42,7 @@ options.registry.progress = options.Class.extend({
 
         // Temporary hide the label. It's effectively removed in cleanForSave
         // if the option is confirmed
-        progressLabel.classList.toggle('d-none', widgetValue === 'none');
+        progressLabel?.classList.toggle("d-none", widgetValue === "none");
```

Comment:
- `Try to change the Label from Hide to 'Display Inside'.` ---> This step is not what causes the traceback.
In this case, `progressLabel` is created in JS because we enter the condition:
`if (!progressLabel && widgetValue !== 'none') {`
The issue actually happens when you hover the `Hide` option while the `progressbar`
is already set to `Hide`. At that moment, we do not enter the condition.
Please update the steps, because when I read them, I could not easily reproduce or understand the bug.